tests/store/api/conftest.py

```diff
@@ -197,13 +198,14 @@ def fixture_rml_pool_store(case_id: str, customer_id: str, helpers, store: Store
         percent_kth=80,
         percent_reads_guaranteed=75,
         sequencing_depth=0,
+        target_reads=800,
     )
     store.add_commit(application)
 
     app_version = store.add_version(
         application=application,
         version=1,
-        valid_from=dt.datetime.today(),
+        valid_from=dt.datetime.now(),
```

Comment:
There is a test fixture for this already. I think it iis called timestamp_now or similar

Reply:
It was called `timestamp_today`. Changed it to now because of sourcery, but I will refactor the fixture to `timestamp_now` instead of `timestamp_today` as it was calling dt.datetime.now()